app/presenters/non_edition_result.rb

```diff
@@ -0,0 +1,60 @@
+class NonEditionResult < SearchResult
+  include ERB::Util
+
+  def to_hash
+    super.merge({
+      metadata: metadata,
+      metadata_any?: metadata.any?,
+    })
+  end
+
+  def format
+    result["document_type"]
+  end
+
+  def metadata
+    data = [
+      last_update,
+      display_type,
+      organisations,
+    ]
+
+    data.select(&:present?)
+  end
+
+private
+  def last_update
+    if result["last_update"]
+      result["last_update"].to_date.strftime("%e %B %Y")
+    end
+  end
+
+  def display_type
+    case format
+    when "aaib_report"
+      "AAIB report"
+    when "cma_case"
+      "CMA case"
+    when "maib_report"
+      "MAIB report"
+    when "raib_report"
+      "RAIB report"
+    else
+      format.humanize
+    end
+  end
+
+  def organisations
+    orgs = Array(result["organisations"]).reject(&:blank?)
```

Comment:
They have the same effect. [`Kernel#Array`](http://ruby-doc.org/core-1.9.3/Kernel.html#method-i-Array) is just a bit more explicit about the intention, and as Edd points out, a little more searchable.